frontend/src/routes/(authenticated)/admin/+page.svelte

```diff
@@ -58,7 +58,12 @@
   $: shownUsers = lastLoadUsedActiveFilter ? users : users.slice(0, 10);
 
   function filterProjectsByUser(user: User): void {
-    $queryParamValues.userEmail = user.email ?? undefined;
+    $queryParamValues.userId = user.id ?? undefined;
```

Comment:
```suggestion
    $queryParamValues.userId = user.id;
```

Reply:
Moot now that I've abandoned the userId approach.